src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -317,8 +317,10 @@ public override IOperation VisitBinaryOperator(IBinaryOperation operation, int?
                 return VisitBinaryConditionalOperator(operation, sense: true, captureIdForResult, fallToTrueOpt: null, fallToFalseOpt: null);
             }
 
-            // PROTOTYPE(dataflow): deal with stack spilling
-            return base.VisitBinaryOperator(operation, captureIdForResult);
+            _evalStack.Push(Visit(operation.LeftOperand));
+            IOperation rightOperand = Visit(operation.RightOperand);
+            SemanticModel model = null;
+            return new BinaryOperatorExpression(operation.OperatorKind, _evalStack.Pop(), rightOperand, operation.IsLifted, operation.IsChecked, operation.IsCompareText, operation.OperatorMethod, model, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
```

Comment:
>model [](start = 196, length = 5)

The change you have made is fine, but named argument refers to an argument of form ```semanticModel: null```, now it can be used in the middle of an argument list.

Reply:
I'd forgotten we added this feature, thanks for reminding me!